ruby/red-arrow/test/test-slicer.rb

```diff
@@ -528,5 +564,48 @@ def setup
 2	window
       TABLE
     end
+
+    test("match_substring?(invalid_pattern)") do
+      assert_raise(ArgumentError) do
```

Comment:
Could you also check error message?

```suggestion
      assert_raise(ArgumentError.new("...")) do
```

Reply:
Done!